cache-config/testing/docker/docker-compose-ats-build.yml

```diff
@@ -26,19 +26,13 @@ volumes:
 services:
 
   trafficserver_build:
-    environment:
-      - ATS_VERSION=8.1.x
-      - CJOSE_URL=https://github.com/cisco/cjose
-      - CJOSE_TAG=latest
-      - JANSSON_URL=https://github.com/akheron/jansson
-      - JANSSON_TAG=v2.11
-      - OPENSSL_URL=https://github.com/openssl/openssl
-      - OPENSSL_TAG=OpenSSL_1_1_1
-      - RUN_ATS_UNIT_TESTS=false
+    env_file:
+      - variables.env
     build:
       context: ../../..
       dockerfile: cache-config/testing/docker/trafficserver/Dockerfile
       args:
-        RHEL_VERSION: ${RHEL_VERSION:-8}
+        - OS_VERSION=${OS_VERSION:-8}
+        - OS_DISTRO=${OS_DISTRO:-rockylinux/rockylinux}
```

Comment:
Rocky Linux now has an official image, `rockylinux/rockylinux` can be just `rockylinux`.